website/project/model.py

```diff
@@ -2359,15 +2371,19 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             user=user,
             foreign_user=foreign_user,
             params=params,
+            node=self,
+            original_node=params['node']
         )
 
         if log_date:
             log.date = log_date
+        log.save()
 
-        self.date_modified = log.date.replace(tzinfo=None)
+        if len(self.logs) == 1:
```

Comment:
The first case is `log.date.replace...` and second is self.logs[-1].date.replace...

My thinking was, if self.logs() equals one, then this add_log method created the first log.  So date_modified and date_created should be the same.  

In this example, I just created a project. `log.date` and `self.log.date` are different, the latter being rounded off.
![screen shot 2016-02-23 at 3 56 53 pm](https://cloud.githubusercontent.com/assets/9755598/13266436/19f50f52-da46-11e5-9a21-d9b786cd3170.png)
![screen shot 2016-02-23 at 3 57 08 pm](https://cloud.githubusercontent.com/assets/9755598/13266438/1b960352-da46-11e5-8d0a-6579f3d1b855.png)

If this is the first log, there should not be a difference between date_created and date_modified, hence the conditional.